### PR TITLE
[PyROOT] Add entry about buffers that point to null

### PR DESCRIPTION
**Pull request by etejedor**
From ticket:

https://sft.its.cern.ch/jira/browse/ROOT-11028

### CODE REVIEW

File: manual/python/index.md [hunk 1]
```diff
@@ -620,3 +620,23 @@ True
...
+>> g = ROOT.TGraph()
+>> x = g.GetX()
+
+>> xl = list(x)  # throws ReferenceError
+```
+
+The code above can be protected by checking first for the validity of `x`:
+
+```python
+>>> if x: xl = list(x)
```
Comment (stwunsch) [inline manual/python/index.md:641]: What about the following?

```suggestion
>>> xl = list(x) if x else 'your default'
```

**Review by stwunsch — APPROVED** [review]
I've added two suggestions, but nothing critical. Looks good!

File: manual/python/index.md (hunk 1 above)
Comment (etejedor) [reply manual/python/index.md:641]: Looks good thanks!

**Comment by etejedor** [comment]
Applied @stwunsch comments, thanks!

**Pull request merged**